team/bundles/org.eclipse.compare/compare/org/eclipse/compare/internal/DocLineComparator.java

```diff
@@ -303,5 +305,15 @@ private boolean compare(String s1, String s2) {
 		}
 		return true;
 	}
+
+	private static boolean isRelevantWhiteSpace(String line, int offset, char character, int lineIndex,
+			DocLineComparator comparator) {
+		boolean result = Character.isWhitespace(character);
```

Comment:
maybe "isWhitespace" is a more speaking name

Reply:
commit 82b15e5

Reply:
sorry, I misunderstood your request. Could you please check commit 43b6196?

Reply:
> sorry, I misunderstood your request. Could you please check commit [43b6196](https://github.com/eclipse-platform/eclipse.platform/commit/43b6196007c7854af08dac22663a84c9ae2457d7)?

I was not precise in my comment. It's not your fault. Looks good now.